pyEDAA/ProjectModel/__init__.py

```diff
@@ -44,6 +44,7 @@
 from pyTooling.Decorators import export
 from pySVModel            import VerilogVersion, SystemVerilogVersion
 from pyVHDLModel          import VHDLVersion
+from pySRDLModel          import SystemRDLVersion
```

Comment:
Are we going to use `pySRDLModel` or `pySystemRDLModel`?

Normally I try to avoid abbreviations, OTOH pySVModel is also shortened because we got problems with headline and logos...

Any recommendation? Is SRDL also a known abbreviation like SV for SystemVerilog?

Reply:
I can see you went with `pySystemRDLModel` and I think that is fine.

Reply:
Please take your decision / suggestion without my created repository. That can be refactored :).

Reply:
```suggestion
from pySystemRDLModel          import SystemRDLVersion
```